src/pages/home/Home.jsx

```diff
@@ -8,80 +8,87 @@ import racquet3 from "../../assets/rq-3.webp";
 import stringImage from "../../assets/string.webp";
 import shuttle from "../../assets/shuttle.webp";
 import { useNavigate } from "react-router-dom";
+import {useFilter} from '../../context/filter-context.js'
 
 const Home = () => {
   const navigate = useNavigate();
+  const {filterDispatch} = useFilter();
+  const featuredCategoriesHandler=(category)=>{
+    filterDispatch({type:"CLEAR_FILTER"});
+      filterDispatch({type:"CATEGORIES",payload:category,isSelected:true});
```

Comment:
I am not sure of it, but I think it is better if you have only two things in dispatch `{type: "CATEGORIES",payload:{category: category,isSelected: true}}`, we should have only two thing is dispatch `type` and `payload` and we should have payload as object.

Reply:
i've changed the action object as you suggested